src/model/configuration_example/training/training_v1.yaml

```diff
@@ -11,4 +11,11 @@ batch_size: 2
 grad: 4
 max_tokens: 2048
 num_shards: 1024
-num_workers: 2
\ No newline at end of file
+num_workers: 2
+epochs: 1
+start_epochs: 0
+start_steps: 0
+save_steps: 10000
+eval_steps: 10000
+save_paths: ./weight
+weight_path: False
```

Comment:
What is the different between weight path and save path. Can you write in comments?

Reply:
I forgot to remove